.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
+build/
+zos.json
```

Comment:
What does this do?

Reply:
/ Do we not need this

Reply:
This was leftover from a previous command I tried running. Suppose is not needed